src/sonic-yang-models/yang-events/sonic-events-bgp.yang

```diff
@@ -0,0 +1,97 @@
+module sonic-events-bgp {
+    namespace "http://github.com/Azure/sonic-events-bgp";
+
+    yang-version 1.1;
+
+    import sonic-events-common {
+        prefix evtcmn;
+    }
+
+    import openconfig-alarm-types {
+        prefix oc-alarm-types;
+    }
+
+    revision 2022-12-01 {
+        description "BGP alert events.";
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONIC BGP events";
+
+    container bgp-state {
```

Comment:
plz add top level container with name same as module

Reply:
YANG review is moved to different [PR #11404 ](https://github.com/Azure/sonic-buildimage/pull/11404)